README.md

```diff
@@ -1,3 +1,5 @@
+> This is the README for the [upcoming major release](CHANGELOG.md#Head). If you have time, please consider [contributing](CONTRIBUTING.md) to the [outstanding tasks](https://github.com/stylelint/stylelint/issues/5205). 
```

Comment:
Removing a trailing whitespace is needed for Prettier 😉 

```suggestion
> This is the README for the [upcoming major release](CHANGELOG.md#Head). If you have time, please consider [contributing](CONTRIBUTING.md) to the [outstanding tasks](https://github.com/stylelint/stylelint/issues/5205).
```

Reply:
Ha, that's what I get for cutting corners and using the GitHub interface to make the change... 😄